src/ApiPort/Resources/LocalizedStrings.resx

```diff
@@ -248,6 +248,7 @@ gives a summary of issues and possible steps forward.
   -r, --resultFormat=VALUE   The report output format
   -p, --showNonPortableApis  Calculate non-portable APIs
   -b, --showBreakingChanges  Calculate breaking changes on full .NET Framework
+  -u, --showRetargettingIssues Include the retargetting issues in the reports  
```

Comment:
I just noticed this offsets the whitespace. Can you adjust the other lines so they look more in line?